packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -279,7 +279,6 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
         uint128 assets,
         address withdrawAccount
     ) external onlyValidatorManager {
-        if (assets == 0) revert NotAllowedZeroInput();
```

Comment:
Do we allow zero asset for registration?

Reply:
It won't happen, since `registerValidator` function at ValidatorManager will check if asset is below the `MIN_REGISTER_AMOUNT`, so it's a duplicated check.